modules/setting/setting.go

```diff
@@ -481,8 +491,8 @@ func NewContext() {
 	}[Cfg.Section("time").Key("FORMAT").MustString("RFC1123")]
 
 	RunUser = Cfg.Section("").Key("RUN_USER").String()
-	// Does not check run user when the install lock is off.
-	if InstallLock {
+	// Does not check run user when the install lock is off or is running tests
+	if InstallLock && AppRunMode != RUN_MODE_TEST {
```

Comment:
I agree. This is a WIP, we should abstract things using interfaces. I am open for suggestions on how to do it right.